symphony/lib/core/class.exceptionhandler.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * @package core
+ */
+
+/**
+ * ExceptionHandler will handle any uncaught exceptions thrown in
+ * Symphony. Additionally, all errors in Symphony that are raised to Exceptions
+ * will be handled by this class.
+ * It is possible for Exceptions to be caught by their own `ExceptionRenderer` which can
+ * provide the `render` function so that it can be displayed to the user appropriately.
+ */
+final class ExceptionHandler
```

Comment:
The entire class is useless, because all properties and methods are static.

I see two options here:

1. remove the class and create real functions under a namespace
2. remove the "static" keyword from all properties and methods; create a real object and inject this object in the classes `Symphony` and `SymphonyExceptionRenderer`

Reply:
The goal is to remove all static methods, thanks for spotting that I can now do it !

Reply:
Done in d4569e1